src/Core/src/Platform/iOS/ApplicationExtensions.cs

```diff
@@ -125,5 +126,29 @@ public static NSUserActivity ToUserActivity(this IPersistedState? state, string
 
 			return userActivity;
 		}
+
+		public static void UpdateUserInterfaceStyle(this IApplication application)
+		{
+			if (application is null)
+				return;
+
+			var currentViewController = WindowStateManager.Default.GetCurrentUIViewController(false);
```

Comment:
If I have multiple windows don't I want to update the theme in all windows and not just the active one?

Reply:
What happens if I have 2 windows?